gradlew

```diff
@@ -109,10 +126,11 @@ if $darwin; then
     GRADLE_OPTS="$GRADLE_OPTS \"-Xdock:name=$APP_NAME\" \"-Xdock:icon=$APP_HOME/media/gradle.icns\""
 fi
 
-# For Cygwin, switch paths to Windows format before running java
-if $cygwin ; then
+# For Cygwin or MSYS, switch paths to Windows format before running java
+if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
```

Comment:
Technically, this has nothing to do with Gradle deprecations. I am not sure what MSYS is, is this the Linux subsystem on Windows?

I suspect the whole file is copied from another project, can you give the source?

Reply:
I just run the normal upgrade command `gradlew wrapper --gradle-version 6.9.1`, nothing special

Reply:
MSYS is something similar to CYGWIN, but not WSL.
So it's cool to have that change in.